ProcessMaker/processmaker
# QA Observation Bugfix/FOUR-17647: Launchpad > `My bookmarks` category has a incorrect message when it is empty

## Solution
- Message for empty bookmark page was added

## How to Test
-Login as admin
-Go to Processes
-Select My Bookmarks in Available processes in left menu
-Deselect all bookmarked processes
-Check Message "Currently, you don’t have any processes bookmarked"

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-17647

## Code Review Checklist
ci:next
